test/with-typescript/index.ts

```diff
@@ -835,6 +835,9 @@ function getSessionFunctions(original: SessionRecipeInterface): SessionRecipeInt
         shouldDoInterceptionBasedOnUrl: function (...input) {
             return original.shouldDoInterceptionBasedOnUrl(...input);
         },
+        getClockSkewInMillis: function (input) {
```

Comment:
Please also add imports for the date provider + a supertokens init call where you define a custom date provider

Reply:
done